test/functional/feature_config_args.py

```
@@ -267,6 +267,15 @@ def test_connect_with_seednode(self):
                     unexpected_msgs=seednode_ignored):
                 self.restart_node(0, extra_args=[connect_arg, '-seednode=fakeaddress2'])
 
+
+    def test_acceptstalefeeestimates_arg_support(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_acceptstalefeeestimates_arg_support`

Add return type annotation: `None`

(ANN201)